proto/iotics/api/meta.proto

```diff
@@ -107,11 +106,48 @@ message SparqlQueryRequest {
 
 // SparqlQueryResponse is a part of a result for a SPARQL query request. Multiple chunks form a complete result. Related
 // chunks can be identified by a combination of:
-// - The hostId
 // - Client reference (in headers, set by caller)
 // - Chunk sequence number
 message SparqlQueryResponse {
 
+  // Payload of the query result chunk
+  message Payload {
+
+    // Position of a chunk in result for a given request. The first chunk has a sequence number of 0.
+    uint64 seqNum = 1;
```

Comment:
Question (CC @StephanieBracaloni ): Do we want to keep this compatible in terms of field indexes by starting with `2` here? (`hostId` was 1 before)?
That way existing SPARQL requests made should still work and `hostId` would be ignored.

Reply:
The field indexes do not matter (ie. it is ok if it starts from 2).
However, we don't want to/don't have to keep things compatible since we have announced a breaking change (even if only internal since sparql query is not used publicly yet) and the response format will also break the compatibility.

Reply:
> The field indexes do not matter 
+
> the response format will also break the compatibility

.. this is the only difference? (I don't understand what you mean but never mind.)

Anyway - if we're OK to break - let's keep start at index `1`, as you would if it were brand new :+1: